pytorch/training/docker/1.13/py3/cu117/Dockerfile.gpu

```diff
@@ -20,6 +20,9 @@ ARG SMD_MODEL_PARALLEL_URL=https://sagemaker-distributed-model-parallel.s3.us-we
 # ZeRO-2D Binary
 ARG ZERO_2D_URL=https://aws-deepspeed-zero-2d-binaries.s3.us-west-2.amazonaws.com/r1.13.1/20230412-070309/4c3ff1af262257f1636b4aafd497e0c332a1bc1d/deepspeed-0.6.1%2B4c3ff1a-py3-none-any.whl
 
+# SMP Binary
+ARG SMP_URL=https://smppy.s3.amazonaws.com/pytorch/cu117/smppy-0.3.325-cp39-cp39-linux_x86_64.whl
```

Comment:
Please use a different/long-form name to prevent confusion with SageMaker Model Parallelism. If not SMPPY, then perhaps "SageMaker Profiler"?